rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/200_dense_vector_docvalue_fields.yml

```diff
@@ -0,0 +1,118 @@
+setup:
+  - requires:
+      cluster_features: "gte_v8.17.0"
+      "test_runner_features": "close_to"
+      reason: 'docvalue_fields parameter for dense_vector fields is required'
+  - do:
+      indices.create:
+        index: test
+        body:
+          mappings:
+            properties:
+              name:
+                type: keyword
+              vector1:
+                type: dense_vector
+                element_type: float
+                dims: 5
+                index: true
+              vector2:
+                type: dense_vector
+                element_type: float
+                dims: 5
+                index: false
+              vector3:
+                type: dense_vector
+                element_type: byte
+                dims: 5
+                index: true
+              vector4:
+                type: dense_vector
+                element_type: byte
+                dims: 5
+                index: false
+              vector5:
+                type: dense_vector
+                element_type: bit
+                dims: 40
+                index: true
+              vector6:
+                type: dense_vector
+                element_type: bit
+                dims: 40
+                index: false
+  - do:
+      index:
+        index: test
+        id: "1"
+        body:
+          name: cow.jpg
+          vector1: [230.0, 300.33, -34.8988, 15.555, -200.0]
+          vector2: [130.0, 115.0, -1.02, 15.555, -100.0]
+          vector3: [-1, 100, -13, 15, -128]
+          vector4: [-1, 50, -1, 1, 120]
+          vector5: [1, 111, -13, 15, -128]
+          vector6: [-1, 11, 0, 12, 111]
+  - do:
+      index:
+        index: test
+        id: "2"
+        body:
+          name: moose.jpg
+
+  - do:
```

Comment:
Let's add a couple of more documents after that with some vector fields missing and some present to ensure correct iteration.

Reply:
Got it, I've added the additional documents to test the iteration.